ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -185,48 +186,92 @@ public struct AvatarGroup: View {
+                            let rtlRingPaddingInterspace = nextAvatarHasRing ? -x + ringGapOffset : -x + ringOffset + (ringOuterGap * 3)
+                            let rtlNoRingPaddingInterspace = nextAvatarHasRing ? -x - ringOffset - ringGapOffset : -x - ringOuterGap
+                            let stackPadding = currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace
+
+                            let xPosition = currentAvatarHasRing ? x - ringGapOffset : x - ringOuterGap
+                            let xPositionRTL = currentAvatarHasRing ? rtlRingPaddingInterspace : rtlNoRingPaddingInterspace
+                            let xOrigin = Locale.current.isRightToLeftLayoutDirection() ? xPositionRTL : xPosition
+                            let yOrigin = sizeDiff / 2
+                            let cutoutSize = isLastDisplayed ? ringGapOffset + imageSize : nextAvatarSize
+
+                            VStack {
+                                avatarView
+                                    .transition(.identity)
+                                    .modifyIf(needsCutout, { view in
+                                        view.mask(Avatar.AvatarCutout(
+                                            xOrigin: xOrigin,
+                                            yOrigin: yOrigin,
+                                            cutoutSize: cutoutSize)
+                                                    .fill(style: FillStyle(eoFill: true)))
+                                    })
+                            }
+                            .padding(.trailing, tokens.style == .stack ? stackPadding : interspace)
+                            .animation(Animation.linear(duration: 0.1))
+                            .transition(AnyTransition.move(edge: .leading))
+                        }
+                        if overflowCount > 0 {
+                            VStack {
+                                createOverflow(count: overflowCount)
+                            }
+                            .transition(AnyTransition.move(edge: .leading))
+                        }
+                    }
+                    .frame(height: geo.size.height,
+                           alignment: .leading)
+                }
+                .frame(width: calcWidth(hasOverflow: overflowCount > 0), height: groupHeight, alignment: .leading)
             }
         }
+
+        return avatarGroupContent
+    }
+
+    private func calcWidth(hasOverflow: Bool) -> CGFloat {
+        var width: CGFloat = 0
+        let maxShown = state.maxDisplayedAvatars < state.avatars.count ? state.maxDisplayedAvatars : state.avatars.count
+        let interspace: CGFloat = state.style == .stack ? tokens.interspace : tokens.interspace
+        let gapOffset: CGFloat = tokens.ringOuterGap * 2
+        let ringGapOffset: CGFloat = tokens.ringThickness + tokens.ringInnerGap + tokens.ringOuterGap
```

Comment:
This offset is being used below on the calculation of the avatar size in case the ring is visible.
Shouldn't these be multiplied by 2 (both sides of the ring thickness, gap and outer gap?

Have you validated the ring scenario?

![Screen Shot 2021-12-09 at 3 25 08 PM](https://user-images.githubusercontent.com/68076145/145491403-f2d3ff14-a609-452e-97a5-7ee285cb4d15.png)
![Screen Shot 2021-12-09 at 3 25 23 PM](https://user-images.githubusercontent.com/68076145/145491422-28053814-4161-49aa-a7f4-bcdea9ee754c.png)

Reply:
In this scenario the Avatars are getting cropped on the left side:

![Screen Shot 2021-12-09 at 3 30 36 PM](https://user-images.githubusercontent.com/68076145/145491896-bf28c27e-1b78-48ff-8ff6-23d9222fcfe9.png)